libsolidity/analysis/ContractLevelChecker.cpp

```diff
@@ -86,6 +86,7 @@ bool ContractLevelChecker::check(ContractDefinition const& _contract)
 
 	checkDuplicateFunctions(_contract);
 	checkDuplicateEvents(_contract);
+	checkReceiveParameters(_contract);
 	m_overrideChecker.check(_contract);
```

Comment:
The problem is that Override Checker won't be able to handle this example.

The ICE happens at

https://github.com/ethereum/solidity/blob/4cbf9ff74c9aabe6047212cb60a28728e66b45aa/libsolidity/analysis/OverrideChecker.cpp#L583

At the following point, the two override functions are set to be compared.

https://github.com/ethereum/solidity/blob/4cbf9ff74c9aabe6047212cb60a28728e66b45aa/libsolidity/analysis/OverrideChecker.cpp#L840